milvus/metadata.csv

```diff
@@ -0,0 +1,387 @@
+milvus.datanode.encode_buffer_latency.bucket,count,,,,Latency of encode buffer data,0,milvus,,,
+milvus.datanode.encode_buffer_latency.count,count,,,,Latency of encode buffer data,0,milvus,,,
+milvus.datanode.encode_buffer_latency.sum,count,,millisecond,,Latency of encode buffer data,0,milvus,,,
+milvus.datanode.flowgraph_num,gauge,,,,Number of flowgraphs,0,milvus,,,
+milvus.datanode.flush.buffer_op.count,count,,,,Count of flush buffer operations,0,milvus,,,
+milvus.datanode.flush.req.count,count,,,,Count of flush request,0,milvus,,,
+milvus.datanode.flushed_data.rows.count,count,,,,Num of rows flushed to storage,0,milvus,,,
+milvus.datanode.flushed_data.size.count,count,,byte,,Byte size of data flushed to storage,0,milvus,,,
+milvus.datanode.msg.dispatcher_tt_lag_ms,gauge,,millisecond,,time.Now() sub dispatcher's current consume time,0,milvus,,,
+milvus.datanode.msg.rows.count,count,,,,Count of rows consumed from msgStream,0,milvus,,,
+milvus.datanode.save_latency.bucket,count,,,,Latency of saving flush data to storage,0,milvus,,,
+milvus.datanode.save_latency.count,count,,,,Latency of saving flush data to storage,0,milvus,,,
+milvus.datanode.save_latency.sum,count,,millisecond,,Latency of saving flush data to storage,0,milvus,,,
+milvus.diskann.range_search_iters.bucket,count,,,,DISKANN range search iterations,0,milvus,,,
+milvus.diskann.range_search_iters.count,count,,,,DISKANN range search iterations,0,milvus,,,
+milvus.diskann.range_search_iters.sum,count,,,,DISKANN range search iterations,0,milvus,,,
+milvus.diskann.search_hops.bucket,count,,,,DISKANN search hops,0,milvus,,,
+milvus.diskann.search_hops.count,count,,,,DISKANN search hops,0,milvus,,,
+milvus.diskann.search_hops.sum,count,,,,DISKANN search hops,0,milvus,,,
+milvus.diskann_bitset_ratio.bucket,count,,,,DISKANN bitset ratio for search and range search,0,milvus,,,
+milvus.diskann_bitset_ratio.count,count,,,,DISKANN bitset ratio for search and range search,0,milvus,,,
+milvus.diskann_bitset_ratio.sum,count,,,,DISKANN bitset ratio for search and range search,0,milvus,,,
+milvus.exec_latency.bucket,count,,,,Execute latency per request,0,milvus,,,
+milvus.exec_latency.count,count,,,,Execute latency per request,0,milvus,,,
+milvus.exec_latency.sum,count,,,,Execute latency per request,0,milvus,,,
+milvus.filter.connectivity_ratio.bucket,count,,,,Avg connectivity ratio set under filtering per request,0,milvus,,,
+milvus.filter.connectivity_ratio.count,count,,,,Avg connectivity ratio set under filtering per request,0,milvus,,,
+milvus.filter.connectivity_ratio.sum,count,,,,Avg connectivity ratio set under filtering per request,0,milvus,,,
+milvus.filter.mv.activated_fields_cnt.bucket,count,,,,Avg mv activated fields per request,0,milvus,,,
+milvus.filter.mv.activated_fields_cnt.count,count,,,,Avg mv activated fields per request,0,milvus,,,
+milvus.filter.mv.activated_fields_cnt.sum,count,,,,Avg mv activated fields per request,0,milvus,,,
```

Comment:
- The description field should explain what the difference is between these three metrics.
- What does "mv" mean?

Reply:
There is no explanation as to what mv means in the Milvus docs.

Reply:
MV seems like is materialized views:
https://github.com/milvus-io/milvus/issues/29892

Reply:
I'm going to assume that it means "materialized view" which is my best guess right now.
[Wikipedia docs](https://en.wikipedia.org/wiki/Materialized_view) on materialized view

Reply:
Didn't mean to ignore you comment @steveny91, it hadn't loaded for me when I commented 🤣